### PR TITLE
Support for JPY and jp-JP

### PR DESCRIPTION
**Pull request by guillermodoghel**
## Description
Implement JPY and jp-JP

## Test plan
Manually tested and extends existing tests

Screenshot 2024-02-28 at 7 09 47 PM
Screenshot 2024-02-28 at 7 39 53 PM
Screenshot 2024-02-28 at 7 50 36 PM
Screenshot 2024-02-28 at 7 54 46 PM
Screenshot 2024-02-28 at 7 54 49 PM
Screenshot 2024-02-28 at 7 55 10 PM
Screenshot 2024-02-28 at 7 58 34 PM
Screenshot 2024-02-28 at 7 58 38 PM

  Example:
   * Unit tests which cover functionality X, Y, Z
   * V was tested via UI tests
   * W was tested manually
-->

### CODE REVIEW

File: packages/client/base/src/types/hosted/index.ts [hunk 1]
```diff
@@ -43,8 +43,8 @@ export type CrossmintPayButtonProps = BaseButtonProps & {
 
 export type CheckoutProps = {
     experimental?: boolean;
-    display: "same-tab" | "new-tab" | "popup";
-    delivery: "custodial" | "non-custodial" | "all";
+    display?: "same-tab" | "new-tab" | "popup";
+    delivery?: "custodial" | "non-custodial" | "all";
```
Comment (pgarciaegido) [inline packages/client/base/src/types/hosted/index.ts:47]: why is this changing?

Comment (guillermodoghel) [reply packages/client/base/src/types/hosted/index.ts:47]: @pgarciaegido this extra mille: https://linear.app/crossmint/issue/PAY-2484/checkout-form-in-react-not-working-when-all-checkoutprops-properties

 These are actually optionals and have default values.

File: packages/client/base/src/types/hosted/index.ts [hunk 2]
```diff
@@ -43,8 +43,8 @@ export type CrossmintPayButtonProps = BaseButtonProps & {
 
 export type CheckoutProps = {
     experimental?: boolean;
```
Comment (mPaella) [inline packages/client/base/src/types/hosted/index.ts:45]: not related, but we should remove this lol

**Review by mPaella — APPROVED** [review]

File: packages/client/base/src/types/hosted/index.ts (hunk 2 above)
Comment (guillermodoghel) [reply packages/client/base/src/types/hosted/index.ts:45]: yeah, but will do on a future release, I want to clean the experimental concept for the whole thing at once (sdk and main)

https://linear.app/crossmint/issue/PAY-2580/remove-the-experimental-flag

**Pull request merged**